django/website/logframe/api.py

```diff
@@ -317,6 +318,16 @@ class RatingViewSet(FilterRelationship, viewsets.ModelViewSet):
     lookup_rel = 'log_frame_id'
 
 
+# Settings
+class SettingsSerializer(ModelSerializer):
+    class Meta:
+        model = Settings
+        fields = (
+            'id',
+            'max_result_level',
+            'open_result_level',
+        )
```

Comment:
Settings are always attached to the logframe, but only settings for the logframe currently being used should be serialized. There is no point in serializing the id of the logframe that the settings are for, so the serializer excludes it.